metadata-ingestion/src/datahub/ingestion/source/looker.py

```diff
@@ -1,3 +1,9 @@
+import sys
+
+# restrict python to <=3.9.9 due to https://github.com/looker-open-source/sdk-codegen/issues/944
+if (sys.version_info.major, sys.version_info.minor, sys.version_info.micro) > (3, 9, 9):
```

Comment:
Can make this slightly simpler:

```suggestion
if sys.version_info > (3, 9, 9):
```